website/public/deploymentguide copy/index.html

```diff
@@ -0,0 +1,579 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0"
+    />
+    <link rel="stylesheet" href="/css/index.css" />
+    <link rel="icon" type="image/png" href="/favicon.png" />
+    <script type="module" src="/js/index.js"></script>
+    <title>CiviForm | Deployment Guide</title>
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-SPPGE89TM8"
```

Comment:
**question:** Confirming this tag is correct? I don't have access to the GA so figured I'd I check.

Reply:
Thanks! I will make sure that we have the right one.